LTS-CHANGELOG.adoc

```diff
@@ -17,6 +17,11 @@ include::content/docs/variables.adoc-include[]
 The LTS changelog lists releases which are only accessible via a commercial subscription.
 All fixes and changes in LTS releases will be released the next minor release. Changes from LTS 1.4.x will be included in release 1.5.0.
 
+[[v1.8.21]]
+== 1.8.21 (TBD)
+
+icon:check[] Core: The name of a Mesh user is now forced to be unique on a database level, to prevent creating users with an already existing username. The duplicated username detection mechanism has also been improved.
```

Comment:
Add CAUTION: comment that duplicate usernames must be removed *before* the update (otherwise Mesh will fail to start)